app/models/miq_report/seeding.rb

```diff
@@ -68,14 +68,15 @@ def sync_from_file(filename, dir, typ)
       rpt[:template_type] = typ
       rec = self.find_by_filename(rpt[:filename])
 
+      type_titleized = typ.titleize
       if rec
-        if rec.filename && (rec.file_mtime.nil? || rec.file_mtime.utc < rpt[:file_mtime])
-          _log.info("#{typ.titleize}: [#{rec.name}] file has been updated on disk, synchronizing with model")
+        if rec.filename && (rec.file_mtime.nil? || rec.file_mtime.utc.to_i < rpt[:file_mtime].to_i)
```

Comment:
> Instead of doing it here, can we do it just once, up on line 64?
> 
> And let's use Time#round -- that'll make our intent clearer. Agree we should still add a comment too, though...
> 
> rpt[:file_mtime] = File.mtime(filename).utc.round # DB and filesystem have different precision

I'll add a comment.
doing the round call only on line 64 will solve it only for the second startup of the server and onward after applying this fix, but I can live with that. :)